### PR TITLE
Adds Hemotoxin

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][feature]

## About the PR 
Adds a new chemical, hemotoxin. Hemotoxin is a blood affecting chemical that causes profuse bleeding, but depletes quickly 1 unit can be made by mixing 1 oil, 1 neurotoxin and 1 sulfuric acid. Fast acting but messy poison, doesnt work well in small amounts.

## Why's this needed? 
Its not really needed its just a "fun" new poison.

## Changelog

```changelog
(u)Hans
(*)Added hemotoxin to the game. Causes bleeding.
```

### CODE REVIEW

File: code/modules/chemistry/Reagents-PoisonEtc.dm [hunk 1]
```diff
@@ -251,6 +251,48 @@ datum
...
+			transparency = 130
+			depletion_rate = 0.6
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if (!M) M = holder.my_atom
+				M.take_toxin_damage(0.5 * mult)
+				take_bleeding_damage(M, null, rand(5,10) * mult, DAMAGE_STAB)
+				if (prob(33))
+					M.emote(pick("groan", "pale", "scream"))
+				if (prob(25))
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-PoisonEtc.dm:271]: you likely want probmult

File: code/modules/chemistry/Reagents-PoisonEtc.dm [hunk 2]
```diff
@@ -251,6 +251,48 @@ datum
...
+			fluid_r = 195
+			fluid_g = 10
+			fluid_b = 10
+			transparency = 180
+			depletion_rate = 1.5
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if (!M) M = holder.my_atom
+				take_bleeding_damage(M, null, 120 * mult, DAMAGE_STAB)
+				if (prob(50))
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-PoisonEtc.dm:291]: probmult

File: code/modules/chemistry/Reagents-PoisonEtc.dm [hunk 3]
```diff
@@ -251,6 +251,48 @@ datum
...
+				take_bleeding_damage(M, null, rand(5,10) * mult, DAMAGE_STAB)
+				if (prob(33))
+					M.emote(pick("groan", "pale", "scream"))
+				if (prob(25))
+					boutput(M, pick("<span class='alert'>You feel like your flesh is melting!</span>", "<span class='alert'>You feel a sharp pain in your chest!</span>"))
+
+				..()
+				return
+
+		harmful/hemotoxinplus
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-PoisonEtc.dm:277]: not a fan of adding a chemical with no way to obtain, seems useless

Comment (colossusqw) [reply code/modules/chemistry/Reagents-PoisonEtc.dm:277]: i can remove it then

File: code/modules/chemistry/Reagents-PoisonEtc.dm (hunk 1 above)
Comment (colossusqw) [reply code/modules/chemistry/Reagents-PoisonEtc.dm:271]: ok i have to be honest this is my first time coding or doing a pr what is the difference between prob and probmult

File: code/modules/chemistry/Reagents-PoisonEtc.dm (hunk 3 above)
Comment (alexeric123) [reply code/modules/chemistry/Reagents-PoisonEtc.dm:277]: Maybe make it possible from artbeakers?

Comment (colossusqw) [reply code/modules/chemistry/Reagents-PoisonEtc.dm:277]: oops, i kinda deleted it...

File: code/modules/chemistry/Reagents-PoisonEtc.dm (hunk 1 above)
Comment (colossusqw) [reply code/modules/chemistry/Reagents-PoisonEtc.dm]: changed to probmult

File: code/modules/chemistry/Reagents-PoisonEtc.dm (hunk 2 above)
Comment (colossusqw) [reply code/modules/chemistry/Reagents-PoisonEtc.dm]: changed to probmult

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 4]
```diff
@@ -4006,3 +4006,12 @@ datum
 			mix_phrase = "The mixture comes together slowly. It doesn't seem like it wants to be here."
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
+
+		hemotoxin
+			name = "hemotoxin"
+			id = "hemotoxin"
+			result = "hemotoxin"
+			required_reagents = list("oil" = 1, "neurotoxin" = 1, "acid" = 1)
+			result_amount = 1
```
Comment (Azrun) [inline code/modules/chemistry/Chemistry-Recipes.dm:4015]: There a balance/thematic/chemistry reason for the yield? Or just scale welding fluid usage?

Comment (colossusqw) [reply code/modules/chemistry/Chemistry-Recipes.dm:4015]: hemotoxin and neurotoxin are common in snake venom, so i tought making one reaction change one snake venom to another would be interesting, oil i put there so chemists can make a small amount with just what they have on lab, but to make larger quantities would require more oil manufacturing.

Comment (Azrun) [reply code/modules/chemistry/Chemistry-Recipes.dm:4015]: I was wondering about the: 3 units in -> 1 unit out.  2/3 of the solution disappears.
Often times that is done some realistic reactions that releases a byproduct (solid/liquid/gas) we don't care about or if some aspect of the reaction requires a catalyst.

There isn't anything necessarily wrong with it, just curious.

Comment (colossusqw) [reply code/modules/chemistry/Chemistry-Recipes.dm:4015]: i could add a byproduct... any ideas?

**Pull request closed without merging**